HotTips/TipObjects/TipObjects.cs

```diff
@@ -68,6 +68,31 @@ public static string GetTipId(string globalTipId)
         }
     }
 
+    public class TipHistoryInfo
+    {
+        public string globalTipId { get; set; }
+
+        public TipLikeEnum tipLikeStatus { get; set; }
+
+        public override string ToString()
+        {
+            return string.Concat(globalTipId, ':', (int)tipLikeStatus);
+        }
+
+        public TipHistoryInfo()
+        {
+
+        }
+        public TipHistoryInfo(string tipHistoryItem)
```

Comment:
Might be best to take two parameters: globalTipId, likeStatus.

Could have another public static method for "ParseTipHistoryItem" that will return a new TipHistoryInfo or raise exception if unparsable.